build.gradle

```diff
@@ -5,26 +5,25 @@ buildscript {
             url 'https://plugins.gradle.org/m2/'
         }
     }
-    dependencies {
-        classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.3.12.RELEASE'
-    }
 }
 
 plugins {
-    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
```

Comment:
I think we should get rid of this but we need a better way to handle dependencies e.g.: https://docs.gradle.org/current/userguide/dependency_management.html#sec:declaring_without_version and be sure of the artifact version we are using.

The version chosen for some don't match what I believe is expected/used currently so that may cause issues. E.g:  `slf4j : '2.0.0-alpha7'`

Please revert the changes in the `build.gradle` that are not directly related with Batch poll changes and let's create another PR in which we address dependency management issues and Gradle config improvements.